ffi/harfbuzz/Cargo.toml

```
@@ -31,15 +31,13 @@ skip_optional_dependencies = true
 denylist = ["bench", "serde", "logging"]
 
 [dependencies]
-harfbuzz-sys = { version = "0.5.0" }
+harfbuzz-traits = { version = "0.6.0" }
 icu_normalizer = { workspace = true }
 icu_properties = { workspace = true }
 icu_provider = {workspace = true, features = ["macros"] }
 displaydoc = { version = "0.2.3", default-features = false }
```

ambivalent on whether to add harfbuzz to the workspace deps, but:

```suggestion
displaydoc = { workpace = true }
```